Regenerate reference docs (#664)
MewsSystems/gitbook-connector-api: operations/agecategories.md

| `ServiceId` | string | required | Unique identifier of [Service](services.md#service) the age category belongs to. |
| `MinimalAge` | integer | optional | Minimal age for the age category. |
| `MaximalAge` | integer | optional | Maximal age for the age category. |
| `Names` | [Localized text](_objects.md#localized-text) | required | All translations of the name of the age category. |
| `ShortNames` | [Localized text](_objects.md#localized-text) | optional | All translations of the short name of the age category. |
| `CreatedUtc` | string | required | Creation date and time of the age category in UTC timezone in ISO 8601 format. |
| `UpdatedUtc` | string | required | Last update date and time of the age category in UTC timezone in ISO 8601 format. |
| `Classification` | [Age category classification](agecategories.md#age-category-classification) | required | The classification of the age category. |

> | `IsActive` | boolean | required | The activity state of the age category. |

Reviewer: Usually this is written like that
```suggestion
| `IsActive` | boolean | required | Whether the age category is still active. |
```
But I would suggest to drop name of the entity and make it generic, this would make it less prone to human error and quicker to document it as it would reference same text. Do a follow up task for that. 

Few examples:
- Whether the entity is still active.
- Activity status of the entity
- Specifies whether the entity is active or inactive

Author: Good point! Fix here: https://github.com/MewsSystems/mews/pull/60350
